examples/find_bbcode_layouts.py

```diff
@@ -50,33 +42,31 @@ def get_best_known_layout_params(
         if code == qldpc.codes.BBCode({x: 6, y: 6}, x**3 + y + y**2, y**3 + x + x**2):
             basis_l = ((1, 1), (1, 2))
             basis_r = ((-1, -1), (-1, -2))
-            shift_lr = (2, 1)
+            shift_lr = (-2, -1)
         elif code == qldpc.codes.BBCode({x: 15, y: 3}, x**9 + y + y**2, 1 + x**2 + x**7):
-            basis_l = ((0, 1), (2, 0))
-            basis_r = ((0, 1), (2, 0))
-            shift_lr = (0, 0)
+            basis_l = ((2, 0), (0, 1))
```

Comment:
It seems as if the best-known plaquette parameters changed, but the objective didn't. I'm guessing this change was just due to some re-order of the search and the first/last best changing, right @perlinm ?

Reply:
Correct, I set these to the first best parameters found by the search.

Note that I also recently changed the best parameters (in a different commit)for one of the codes because there were better parameters found when changing `DEFAULT_PRECISION` from 0.1 to 0.01 (which determines when the bisection minimizer quits).